controller/taskController.js

```diff
@@ -0,0 +1,110 @@
+const { task } = require('../models/index')
+
+class taskController {
+   static getAll (req, res, next) {
+      task.findAll()
+      .then(taskData => {
+         res.status(200).json({task : taskData})
```

Comment:
```suggestion
         res.status(200).json({tasks : taskData})
```
soalnya kan getAll berarti lebih dari 1 task , jadi tasks